### PR TITLE
Add a README for the enumerate_github command.

### PR DESCRIPTION
**Pull request by calebbrown**

### CODE REVIEW

File: cmd/enumerate_github/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+## Usage
+
+```shell
+$ enumerate_github [FLAGS]... FILE
+```
+
+The URL for each repository is written to `FILE`. If `FILE` is `-` the results will be written to STDOUT.
+
+### Flags
```
Comment (oliverchang) [inline cmd/enumerate_github/README.md:32]: Are any flags required vs optional? Might be good to add a line on that.

File: cmd/enumerate_github/README.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  changes during enumeration.
+
+#### Misc Flags
+
+- `-log level` set the level of logging. Can be `debug`, `info` (default), `warn` or `error`.
+- `-workers int` the total number of concurrent workers to use. Default is `1`.
+
+### Authentication
+
+Expects an environment variable with one or more GitHub Personal Access Tokens,
```
Comment (oliverchang) [inline cmd/enumerate_github/README.md:69]: Maybe more clearly worded as "A comma delimited environment variable with one or more GitHub Personal Access Tokens must be set" ?

**Review by oliverchang — APPROVED** [review]
thanks!

File: cmd/enumerate_github/README.md (hunk 1 above)
Comment (calebbrown) [reply cmd/enumerate_github/README.md:48]: Added.

File: cmd/enumerate_github/README.md (hunk 2 above)
Comment (calebbrown) [reply cmd/enumerate_github/README.md]: Done. I also moved this section above the Flags documentation.

**Review by oliverchang — APPROVED** [review]

**Pull request merged**